cmp_benchmarks.sh

```diff
@@ -17,9 +17,12 @@ usage="Usage: ./cmp_bnchm.sh -m mode -n name_tag -b baseline -c contender \
     \n\
     Optional arguments:\n\
     -f : a threshold [0, 1] for filtering significant results, e.g. 0.2 . Defaults to 0.05 (5% difference).\n\
-    -o : full/path/to/save/results . Defaults to current directory.\n"
+    -o : full/path/to/save/results . Defaults to current directory.\n\
+    -j : number of jobs. This number is passed to make. Defaults to -1.\n"
```

Comment:
`-j : number of jobs. This number is passed to make. Defaults to 1.\n"`

(Does not match the default value below)

Reply:
Yes, I notice. I fixed this in one of the other PRs. Hope that works for you